pkg/monitoring/metrics/operator_metrics.go

```diff
@@ -0,0 +1,181 @@
+package metrics
+
+import (
+	"strings"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	ioprometheusclient "github.com/prometheus/client_model/go"
+)
+
+const (
+	counterLabelCompName = "component_name"
+	counterLabelAnnName  = "annotation_name"
+
+	HyperConvergedExists    = float64(1)
+	HyperConvergedNotExists = float64(0)
+
+	SingleStackIPv6True = float64(1)
+)
+
+const (
+	SystemHealthStatusHealthy float64 = iota
+	SystemHealthStatusWarning
+	SystemHealthStatusError
+)
+
+var (
+	overwrittenModifications = newCounterVec(
+		"kubevirt_hco_out_of_band_modifications_total",
+		"Count of out-of-band modifications overwritten by HCO")
+	unsafeModifications = newGaugeVec(
+		"kubevirt_hco_unsafe_modifications",
+		"Count of unsafe modifications in the HyperConverged annotations")
+	HyperConvergedCRExists = newGauge(
+		"kubevirt_hco_hyperconverged_cr_exists",
+		"Indicates whether the HyperConverged custom resource exists (1) or not (0)")
+	systemHealthStatus = newGauge(
+		"kubevirt_hco_system_health_status",
+		"Indicates whether the system health status is healthy (0), warning (1), or error (2), "+
+			"by aggregating the conditions of HCO and its secondary resources")
+	singleStackIpv6 = newGauge(
+		"kubevirt_hco_single_stack_ipv6",
+		"Indicates whether the underlying cluster is single stack IPv6 (1) or not (0)")
```

Comment:
this is not an operator metrics as the filename suggests 'metrics/operator_metrics.go', please create a new file with this metric

Reply:
Done